src/components/NFTArticle/SlateEditor/Plugins/SlateImagePlugin.ts

```diff
@@ -88,7 +103,13 @@ export const withImages = (
               at: childPath
             })
           }
-        }
+	} else {
+	  // any child that is not the first image in the figure moved after the figure
+	  if (!(child.type === 'image' && childPath[childPath.length - 1] === 0)) {
+	    Transforms.liftNodes(editor, {at: childPath})
+	    return;
+	  }
+	}
```

Comment:
@ciphrd I added this line which moves any child beside the first image after the figure. We could also change the `liftNodes` to `removeNodes` but imho the UX is better when e.g. the images are actually pasted into the editor. What do you say?

Reply:
Yes I believe this is a great UX, perfect :)